sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/IntegrationTestBase.java

```diff
@@ -66,8 +66,9 @@
     private String testName;
     private final Scheduler scheduler = Schedulers.parallel();
 
-    private static final byte[] CONTENTS_BYTES = "Some-contents".getBytes(StandardCharsets.UTF_8);
+    protected static final byte[] CONTENTS_BYTES = "Some-contents".getBytes(StandardCharsets.UTF_8);
     protected String sessionId;
+    protected String sessionId2;
```

Comment:
These one off sessionIds or variables you shouldn't be putting into the base class. It pollute the base class. They should be in the test method itself where it may be used once.